Wikipedia/Localizations/ksh.lproj/Localizable.strings

```diff
@@ -287,8 +287,10 @@
 "nearby-distance-label-miles" = "$1 Meile";
 "nearby-pull-to-refresh-prompt" = "Trick zom neu Aanzeije";
 "nearby-pull-to-refresh-is-refreshing" = "Neu aam Lahde";
-"nearby-loading" = "Loading nearby articles...";
-"nearby-loaded" = "Nearby articles loaded";
+// Fuzzy
+"nearby-loading" = "Loading nearby articles...<!-- https://phabricator.wikimedia.org/T135832 -->";
```

Comment:
@montehurd, I wasn't sure if you wanted these comments included or not but figured they needed to be communicated. If you don't want them, please feel free to add another commit or make the change in TWN and I'll resync and force push.